starters/deno-oak-denodb/src/__tests__/mutations_test.ts

```diff
@@ -0,0 +1,95 @@
+import {
+	afterEach,
+	assertEquals,
+	beforeEach,
+	describe,
+	it,
+	Spy,
+	spy,
+	Stub,
+	stub,
+} from '../../dev_deps.ts';
+import {
+	createTechnology,
+	deleteTechnologyById,
+	updateTechnology,
+} from '../graphql/resolvers/mutation_handler.ts';
+import { TechnologyRepository } from '../db/repository/technology_repository.ts';
+
+describe('createTechnology', () => {
+	let repositoryCreateStub: Stub, repositoryUpdateStub: Stub, repositoryDeleteByIdStub: Stub;
+
+	const MOCK_CREATE_INPUT = {
+		id: '1',
+		displayName: 'technology1',
+		description: 'whatever',
+		url: 'http://lulz.com',
+	};
+
+	beforeEach(() => {
+		repositoryCreateStub = stub(
+			TechnologyRepository,
+			'create',
+			() => Promise.resolve(MOCK_CREATE_INPUT as any),
+		);
+
+		repositoryUpdateStub = stub(TechnologyRepository, 'update', () =>
+			Promise.resolve({
+				displayName: 'technology2',
+				description: 'whatever',
+				url: 'http://lulz.com',
+			} as any));
+
+		repositoryDeleteByIdStub = stub(
+			TechnologyRepository,
+			'deleteById',
+			() => Promise.resolve(MOCK_CREATE_INPUT as any),
+		);
+	});
+
+	afterEach(() => {
+		repositoryCreateStub.restore();
+		repositoryUpdateStub.restore();
+		repositoryDeleteByIdStub.restore();
+	});
+
+	it('creates a technology', async () => {
+		const MOCK_CACHE: { writeItem: Spy; invalidateItem: Spy } = {
```

Comment:
Maybe we can create a type for this:  `type MockCache = { writeItem: Spy; invalidateItem: Spy }` at the top of the file so it would reduce the reading overhead a little in tests. Just a suggestion though

Reply:
Yeah, good idea.